### PR TITLE
Kibana Input Plugin: added compatibility for all 6.X.X versions

### PR DESCRIPTION
**Pull request by vikkyomkar**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

**Comment by vikkyomkar** [comment]
this PR fixes #6894 and #5744

**Comment by vikkyomkar** [comment]
@danielnelson can we get some eyes on this PR ?

**Comment by lpic10** [comment]
With these changes the plugin seems to work up to version 7.5 (there are no changes to these metrics even in master branch: https://github.com/elastic/kibana/blob/master/src/legacy/server/status/lib/metrics.js)

@vikkyomkar The change on kibana that needs the fix happened on version 6.4, not 6.5, so it would be better to change the PR to reflect that. 

Also, there is a metric name change, but the underlying metric seems to be the same:
https://github.com/elastic/kibana/blob/cb16be83a47c5bc8a5214c2461dddb8c2299f6f5/src/server/status/metrics.js#L32
https://github.com/elastic/kibana/blob/564b25f2ab7b33545e38980443c24162f6ad6d4e/src/server/status/lib/metrics.js#L74

I don't know if it is a good practice to change the metric name on telegraf. OTOH I think the metric name in the updated kibana makes more sense.

@danielnelson what do you recommend?

File: plugins/inputs/kibana/kibana.go [hunk 1]
```diff
@@ -187,15 +199,27 @@ func (k *Kibana) gatherKibanaStatus(baseUrl string, acc telegraf.Accumulator) er
...
-
-	fields["uptime_ms"] = kibanaStatus.Metrics.UptimeInMillis
 	fields["concurrent_connections"] = kibanaStatus.Metrics.ConcurrentConnections
-	fields["heap_max_bytes"] = kibanaStatus.Metrics.Process.Mem.HeapMaxInBytes
-	fields["heap_used_bytes"] = kibanaStatus.Metrics.Process.Mem.HeapUsedInBytes
 	fields["response_time_avg_ms"] = kibanaStatus.Metrics.ResponseTimes.AvgInMillis
 	fields["response_time_max_ms"] = kibanaStatus.Metrics.ResponseTimes.MaxInMillis
 	fields["requests_per_sec"] = float64(kibanaStatus.Metrics.Requests.Total) / float64(kibanaStatus.Metrics.CollectionIntervalInMilles) * 1000
 
+	Version, err := strconv.ParseFloat(strings.Join(strings.Split(kibanaStatus.Version.Number, ".")[:2], "."), 64)
```
Comment (danielnelson) [inline plugins/inputs/kibana/kibana.go]: This line could panic with unexpected version info, in particular when indexing `[:2]`.  Check the length of the split item to ensure it is long enough.  To match Go naming convention use a lowercase variable `version :=`.

File: plugins/inputs/kibana/kibana.go [hunk 2]
```diff
@@ -187,15 +199,27 @@ func (k *Kibana) gatherKibanaStatus(baseUrl string, acc telegraf.Accumulator) er
...
+	if err != nil {
+		return err
+	}
+
+	if Version >= 6.5 {
+		fields["uptime_ms"] = kibanaStatus.Metrics.Process.UptimeInMillis
+		fields["heap_total_bytes"] = kibanaStatus.Metrics.Process.Memory.Heap.TotalInBytes
```
Comment (danielnelson) [inline plugins/inputs/kibana/kibana.go]: For backwards compatibility, either record this only as only `heap_max_bytes` or as both `heap_total_bytes` and `heap_max_bytes`.  Make sure to add a comment so we can remember why we did this.

File: plugins/inputs/kibana/README.md [hunk 3]
```diff
@@ -52,12 +52,16 @@ with following rules:
...
 
 ### Example Output
+```
+Kibana Version < 6.5
+kibana,host=myhost,name=my-kibana,source=localhost:5601,status=green,version=6.3.2 concurrent_connections=0i,heap_max_bytes=136478720i,heap_used_bytes=119231088i,requests_per_sec=1,response_time_avg_ms=59i,response_time_max_ms=300i,status_code=1i,uptime_ms=2187428019i 1534864502000000000
 
-kibana,host=myhost,name=my-kibana,source=localhost:5601,version=6.3.2 concurrent_connections=0i,heap_max_bytes=136478720i,heap_used_bytes=119231088i,response_time_avg_ms=0i,response_time_max_ms=0i,status="green",status_code=1i,uptime_ms=2187428019i 1534864502000000000
+Kibana Version > 6.5
+kibana,host=myhost,name=my-kibana,source=localhost:5601,status=green,version=6.5.4 concurrent_connections=8i,heap_total_bytes=447778816i,heap_used_bytes=380603352i,requests_per_sec=1,response_time_avg_ms=57.6,response_time_max_ms=220i,status_code=1i,uptime_ms=6717489805i 1534864502000000000
+```
```
Comment (danielnelson) [inline plugins/inputs/kibana/README.md]: Only show the >6.5 version

File: plugins/inputs/kibana/kibana.go (hunk 2 above)
Comment (vikkyomkar) [reply plugins/inputs/kibana/kibana.go]: code updated along with the comments as per the suggestions

File: plugins/inputs/kibana/kibana.go (hunk 1 above)
Comment (vikkyomkar) [reply plugins/inputs/kibana/kibana.go]: @danielnelson code is updated to handle any panic situation

File: plugins/inputs/kibana/README.md (hunk 3 above)
Comment (vikkyomkar) [reply plugins/inputs/kibana/README.md]: Plugin output is made generic for all the versions

**Pull request merged**